doc/next-steps/filter-examples.md/$gt.md

```diff
@@ -0,0 +1,76 @@
+
+# $gt
+
+Records that are greater than the given value will be returned.
```

Comment:
can we mention how this works on strings? how does it work with case sensitivity?